multi-server-install-intro.adoc

```diff
@@ -118,10 +118,12 @@ These are ports typically only used by the Zimbra system itself.
 |7993|imaps|mailbox|Backend IMAP over TLS (if proxy configured); zimbraImapSSLBindAddress
 |7995|pop3s|mailbox|Backend POP3 over TLS (if proxy configured); zimbraPop3SSLBindAddress
 |8080|http|mailbox|Backend HTTP (if proxy configured on same host); zimbraMailBindAddress
+|8143|imap|imapd|IMAPD server running independent of the mailboxd process
 |8443|https|mailbox|Backend HTTPS (if proxy configured on same host); zimbraMailSSLBindAddress
 |8465|milter|mta/opendkim|OpenDKIM milter service; localhost
 |8735|zextras|mailbox|internal mailbox to mailbox communication.
 |8736|zextras|mailbox|distributed configuration
+|8993|imap|imapd|IMAPD server running independent of the mailboxd process (encrypted)
```

Comment:
nit: prefer something like: IMAP over TLS via IMAPD server running independent of the mailboxd process